src/librustdoc/html/static/main.js

```diff
@@ -1,9 +1,8 @@
 // From rust:
```

Comment:
Seems this comment is also unnecessary now?

Reply:
Absolutely!